storybookjs/design-system
# Pull (most of) Footer from `frontpage` into the DS

Fixes https://linear.app/chromaui/issue/CH-708/pull-common-page-footer-from-frontpage-into-ds

## Description

Pulling the Footer component from https://github.com/storybookjs/frontpage into the DS in preparation for future work.

### Of Note

- Following [this proposal](https://www.notion.so/Styling-solutions-across-design-system-and-its-consumers-aba73999c0114448ad258ae4f0a24010) and branching off of the branch converting the DS to Emotion.
- I've left off the email subscription for the time being and created [an issue](https://linear.app/chromaui/issue/CH-801/pull-confirmed-mailing-list-from-frontpage-into-ds) to pull that into the DS as well.


### QA instructions

Confirm the styles (mostly) match [the real thing](https://storybook.js.org/).

<details>
  <summary>📦 Published PR as canary version: <code>6.4.2-canary.314.b40307e.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/design-system@6.4.2-canary.314.b40307e.0
  # or 
  yarn add @storybook/design-system@6.4.2-canary.314.b40307e.0
  ```
</details>
